test/inductor/test_memory_planning.py

```diff
@@ -24,6 +24,17 @@
 from torch.export import Dim
 from torch.utils._triton import has_triton
 
+try:
+    try:
+        from .test_aot_inductor import AOTIRunnerUtil
+    except ImportError:
+        from test_aot_inductor import AOTIRunnerUtil
+except (unittest.SkipTest, ImportError) as e:
+    sys.stderr.write(f"{type(e)}: {e}\n")
+    if __name__ == "__main__":
+        sys.exit(0)
```

Comment:
What's the except for? If "this file should run even without AOTInductor" then we can probably put the try-except into `test_abi_compatible` ?